.github/workflows/ci.yml

```
@@ -0,0 +1,252 @@
+        if: failure()
+        with:
+          name: playwright-traces-${{ matrix.node-version }}
+          path: |
+            packages/frontend/test-results/
+            packages/frontend/playwright-report/
+            **/trace.zip
+
+      # Coverage upload (nur für Node 20)
+      - name: Upload coverage to Codecov
+        if: matrix.node-version == 20
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)